memcpy_sse41.c

```diff
@@ -114,9 +114,14 @@ int memcpy_uncached_load_sse41(void *dest, const void *src, size_t n_bytes)
             n -= sizeof(__m128i);
         }
     }
-    _mm_sfence();
+
     if (n)
         memcpy(d, s, n);
+
```

Comment:
@spotluri I guess this comment is incorrect, as when writing to host memory the write fence is needed because of the NT stores.